configs/vid/temporal_roi_align/README.md

```diff
@@ -0,0 +1,28 @@
+# Temporal RoI Align for Video Object Recognition
+
+## Introduction
+
+[ALGORITHM]
+
+```latex
+@inproceedings{gong2021temporal,
+  title={Temporal ROI Align for Video Object Recognition},
+  author={Gong, Tao and Chen, Kai and Wang, Xinjiang and Chu, Qi and Zhu, Feng and Lin, Dahua and Yu, Nenghai and Feng, Huamin},
+  booktitle={Proceedings of the AAAI Conference on Artificial Intelligence},
+  volume={35},
+  number={2},
+  pages={1442--1450},
+  year={2021}
+}
+```
+
+## Results and models on ImageNet VID dataset
+
+We observe around 0.5 mAP fluctuations in performance, and provide the best model.
```

Comment:
We observed that the performance of this method has a fluctuation of about 0.5 mAP. The checkpoint provided below is the best one from all experiments.